pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/SparkSegmentMetadataPushIntegrationTest.java

```diff
@@ -53,6 +53,8 @@
 public class SparkSegmentMetadataPushIntegrationTest extends BaseClusterIntegrationTest {
 
   private JavaSparkContext _sparkContext;
+  private final String _testTable = DEFAULT_TABLE_NAME;
```

Comment:
batch segment upload API requires the table type, otherwise the api wouldn't work.
this should be note in the pinot docs.